.github/ISSUE_TEMPLATE/story.md

```diff
@@ -34,11 +34,9 @@ As a _, so that _, I need _.
     - [ ] Main `README.md`
     - [ ] Other READMEs in the repo
     - [ ] If applicable, update the `ReportStream Setup` section in `README.md`
-  - [ ] Threat model updated
+  - [ ] [Threat model](https://lucid.app/lucidchart/8c6e8d37-2612-42a8-ac57-150c83f8e29e/edit) updated
```

Comment:
Can anyone with access to this link edit it?

Reply:
I don't think so

Reply:
Correct, that isn't a public link.  It works if you are logged in, but not if you aren't logged in.